### PR TITLE
Update Integration pages layout

### PR DESCRIPTION
**Pull request by iArchitSharma**
**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 732e18eb556120a5b2cccb2484bf414d60fea6ad at: https://65bd0927df4b2f009f651073--meshery-docs-preview.netlify.app

File: docs/_layouts/integration.html [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+    <li>{{ feature }}</li>
+
+    {% endfor %}
+</ul>
+
+<h2>How it Works</h2>
+<h6>{{ page.howItWorks }}</h6>
+<p>{{ page.howItWorksDetails }}</p>
+
+<p>To get started with {{ page.title }}, follow these simple steps: <a href="/installation">See Installation Guide</a></p>
```
Comment (leecalcote) [inline docs/_layouts/integration.html:25]: Call me when you're next at keyboard, @iArchitSharma. You need a different format; different set of data here.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/_layouts/integration.html (hunk 1 above)
Comment (leecalcote) [reply docs/_layouts/integration.html:25]: @vishalvivekm @ritiksaxena124 we're not trying to have two sites with the same set of page information.

**Comment by vishalvivekm** [comment]
@iArchitSharma 
Let's discuss this on website call at 6:30 PM IST / 7 AM Central time Today. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

File: docs/_layouts/integration.html (hunk 1 above)
Comment (leecalcote) [reply docs/_layouts/integration.html:25]: @iArchitSharma where are you at on this?

Comment (leecalcote) [reply docs/_layouts/integration.html:25]: Anywhere?

**Pull request closed without merging**